jekyll/_cci2/configuration-reference.adoc

```diff
@@ -2633,6 +2637,55 @@ Jobs are run concurrently by default, so you must explicitly require any depende
 
 '''
 
+[#upstream]
```

Comment:
Isn't this the internal representation used after we compile the config?
I believe we have to update the `requires` field

Reply:
Yes